src/classes/UTIL_Currency.cls

```diff
@@ -149,20 +149,154 @@ public without sharing class UTIL_Currency implements UTIL_Currency.Interface_x
      * @description Mapping of common currency ISO codes to currency symbols
      */
     private static Map<String, String> currencySymbols = new Map<String, String>{
-        'USD' => '$',
+        'AED' => 'د.إ',
+        'AFN' => '؋',
+        'ALL' => 'L',
+        'AMD' => '֏',
+        'ANG' => 'ƒ',
+        'AOA' => 'Kz',
+        'ARS' => '$',
+        'AUD' => '$',
+        'AWG' => 'ƒ',
+        'AZN' => '₼',
+        'BAM' => 'KM',
+        'BBD' => '$',
+        'BDT' => '৳',
+        'BGN' => 'лв',
+        'BHD' => '.د.ب',
+        'BIF' => 'FBu',
+        'BMD' => '$',
+        'BND' => '$',
+        'BOB' => '$b',
+        'BRL' => 'R$',
+        'BSD' => '$',
+        'BTN' => 'Nu.',
+        'BWP' => 'P',
+        'BYN' => 'Br',
+        'BZD' => 'BZ$',
+        'CAD' => '$',
+        'CDF' => 'FC',
+        'CHF' => 'CHF',
+        'CLP' => '$',
         'CNY' => '¥',
-        'JPY' => '¥',
+        'COP' => '$',
+        'CRC' => '₡',
+        'CSD' => 'CSD'
```

Comment:
@Julian88Tex A comma is missing here. That's why the builds are failing for this branch.

Reply:
@force2b oops I think I CMD+Z'd a one times too many before the commit. corrected that thanks!